src/app/applications/iot-devices/downlink.model.ts

```diff
@@ -1,6 +1,6 @@
 
 export class Downlink {
     data: string;
-    port = 0;
-    confirmedDownlink = false;
+    port: number = 0;
```

Comment:
Redundant type as it's inferred from the default value.
![image](https://user-images.githubusercontent.com/92869496/145428146-d5304c87-726c-475a-816e-24d73063ff51.png)